### ISSUE TITLE
Enum argument to MemberData method improperly triggers xUnit1035

### ISSUE THREAD

**Issue by credfeto** [xunit/xunit#2817]
Just updated to xunit.analyzers 1.5.0 from 1.4.0 and some test that previously compiled cleanly now report errors.

Enums passed to the test method specified in ``MemberData`` atributes are reported with with the  xUnit1035 diagnostic e.g:

```
ExampleTests.cs(9, 17): [xUnit1035] The value is not convertible to the method parameter 'networkSelection' of type 'Example.NetworkSelection'. Use a compatible data value. (https://xunit.net/xunit.analyzers/rules/xUnit1035)
```

e.g
```csharp
[MemberData(nameof(NetworkConfigurationBuilder.Build),
                NetworkSelection.SEPOLIA,
                EndpointValidity.VALID,
                NetworkNodeProtocol.HTTPS,
                NetworkNodeType.ONE_NODE_IS_MULTI,
                MemberType = typeof(NetworkConfigurationBuilder))]
```

with method:

```
public static IEnumerable<object[]> Build(
     NetworkSelection networkSelection, 
     EndpointValidity endpointValidity,
     NetworkNodeProtocol protocol,
     NetworkNodeType fullNodeType)
```

each of ``NetworkSelection``, ``EndpointValidity``, ``NetworkNodeProtocol``, ``NetworkNodeType`` are enums

Example Repo: 

https://github.com/credfeto/xunit-analyzers-1-5-0-xUnit1035

Let me know if you need more information

**Comment by bradwilson** [comment]
Available in `1.6.0-pre.3`.

**Comment by credfeto** [comment]
Thanks
